e2e/test_modules/with_tfvars_file/main.tf

```diff
@@ -0,0 +1,15 @@
+terraform {
+  required_providers {
+    # Provider source is used for Terraform discovery and installation of
+    # providers. Declare source for all providers required by the module.
+    local = {
+      source  = "hashicorp/local"
+      version = ">= 2.1.0"
+    }
+  }
+}
+
+resource "local_file" "obj" {
+  content  = join(",", var.sample.tags, [for key, value in var.sample.user_meta : "${key}=${value}"])
```

Comment:
Looks like this module uses the tfvars file object in the resource. it looks like the e2e test checks `variables.auto.tfvars` but not the resources. do you think it would be add value to add the text file contents?

Reply:
I don't know if it would add much for this particular case which is just testing that things are hooked up correctly for the CLI.

This module was from my own testing, doing this check might be useful as we expand the e2e testing for other cases? I'll make a note of it in our task lifecycle e2e testing asana task

Reply:
I see, adding that check is more exercising the Terraform-side of CTS. For me it makes it a little more "to the end" in the e2e-ness of the test, if that makes sense. But, it seems subjective. I'm ok with whatever you decide. Keeping note of it works for me!